react/QuickOrder.tsx

```diff
@@ -159,8 +159,8 @@ const QuickOrder: StorefrontFunctionComponent<QuickOrderProps &
   const addToCartCopyNPaste = () => {
     const items: any = reviewItems.map(({ vtexSku, quantity }: any) => {
       return {
-        id: vtexSku,
-        quantity,
+        id: parseInt(vtexSku, 0),
```

Comment:
The second argument is the base, should be 10 or omit it
https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/parseInt